.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "Documentation"
+about: Report an issue related to the NumPy documentation
```

Comment:
Can we make this auto-add the documentation label(s)?

Reply:
```suggestion
about: Report an issue related to the NumPy documentation
labels: 04 - Documentation
```
should do the trick

Reply:
Great idea, thanks!